### PR TITLE
Azure CLI commands for communication

### PR DESCRIPTION
**Pull request by ravithanneeru**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by kairu-ms** [comment]
@ravithanneeru could you pull the latest code and commit again?

**Comment by moirf** [comment]
> @ravithanneeru could you pull the latest code and commit again?

@kairu-ms latest code taken and all checks passing now.

File: src/communication/azext_communication/manual/_help.py [hunk 1]
```diff
@@ -111,3 +111,58 @@
...
+    type: command
+    short-summary: "Issues a new access token with the specified scopes for a given User Identity. If no User Identity is specified, creates a new User Identity as well."
+    examples:
+      - name: issue-access-token
+        text: |-
+               az communication identity issue-access-token --scope chat
+      - name: issue-access-token with multiple scopes and userid
+        text: |-
+               az communication identity issue-access-token --scope chat voip --userid "8:acs:xxxxxx"
+"""
```
Comment (kairu-ms) [inline src/communication/azext_communication/manual/_help.py:130]: What's this command used for? Where will user use the access token?

Comment (moirf) [reply src/communication/azext_communication/manual/_help.py:130]: @kairu-ms This command is used for generate an access_token can be used to make call to communication service endpoints. This command help achieve this in azure cli - [Quickstart: Create and manage access tokens](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/access-tokens?pivots=programming-language-csharp)

Comment (moirf) [reply src/communication/azext_communication/manual/_help.py:130]: @kairu-ms can you please review?

File: src/communication/azext_communication/manual/_client_factory.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+# pylint: disable=unused-argument
+def cf_communication_identity(cli_ctx, kwargs):
```
Comment (lsundaralingam) [inline src/communication/azext_communication/manual/_client_factory.py:8]: What's ```cli_ctx``` used for?

**Review by kairu-ms — APPROVED** [review]

File: src/communication/azext_communication/manual/_help.py (hunk 1 above)
Comment (kairu-ms) [reply src/communication/azext_communication/manual/_help.py:130]: @moirf, If you want to release this extension, you should update the version of the extension and add release history

Comment (moirf) [reply src/communication/azext_communication/manual/_help.py:130]: @kairu-ms version and release history updated.

**Pull request merged**